scripts/resetLastBlock.js

```diff
@@ -0,0 +1,60 @@
+const Redis = require('ioredis')
+const path = require('path')
+require('dotenv').config({
+  path: path.join(__dirname, '../.env')
+})
+
+const redis = new Redis(process.env.REDIS_URL)
+
+redis.on('error', () => {
+  logError('Error: Cannot connect to redis')
+})
+
+if (process.argv.length < 4) {
+  logError(
+    'Please provide process key and new block value. Example:' +
+      '\n  signature-request 12345 ' +
+      '\n  collected-signatures 12345 ' +
+      '\n  affirmation-request 12345'
+  )
+}
+
+function logError(message) {
+  console.log(message)
+  process.exit(1)
+}
+
+function getRedisKey(name) {
+  const isErcToErc = process.env.BRIDGE_MODE && process.env.BRIDGE_MODE === 'ERC_TO_ERC'
+  const prefix = isErcToErc ? 'erc-' : ''
+  return `${prefix}${name}:lastProcessedBlock`
+}
+
+async function main() {
+  try {
+    const processName = process.argv[2]
+    const newBlockValue = process.argv[3]
+    const lastBlockRedisKey = getRedisKey(processName)
+
+    const value = await redis.get(lastBlockRedisKey)
+
+    if (!value) {
```

Comment:
Notice that this won't allow setting a value for a key that hasn't been initialized yet.

Reply:
I think it's ok in this case. The script is intended for updating the existing values to re-process past events. On the first run, when keys are not initialized yet, the watcher will use `HOME_START_BLOCK` and `FOREIGN_START_BLOCK` from `.env`